tauri-app/src-tauri/src/steam_manager.rs

```diff
@@ -8,6 +9,22 @@ pub struct SteamManager {
 }
 
 impl SteamManager {
+    pub fn get_game_path(path: &Path, app_id: u32) -> PathBuf {
```

Comment:
you wont need to pass in `path` because SteamManager already has `self.home`, which is all you are using it for

Reply:
Ah, I was still trying to figure out how to use that.